### ISSUE TITLE
Head type la-num-adj missing

### ISSUE THREAD

**Issue by fpw** [tatuylonen/wiktextract#49]
The head type `la-num-adj` is not exported, seen for example on this page: https://en.wiktionary.org/wiki/unus#Latin (the numeral entry).

I believe this is because the `head_tag_re` regex misses `num-adj`.

**Comment by fpw** [comment]
la-pronoun is also missing, both fixed in #51

**Comment by fpw** [comment]
I finally found a list of headword template: https://en.wiktionary.org/wiki/Category:Latin_headword-line_templates
This should be the last update to the PR :)

**Comment by tatuylonen** [comment]
Thanks for the patch and report!  Your patch looked generally good, but I ended up reimplementing it differently.  I found dozens of other templates that are used in the head position and changed how that regular expression is constructed.  (There are probably still others that I missed, that invoke the "head" template indirectly)

I just committed the changes and they should be reflected on https://kaikki.org tomorrow (unless something goes wrong with the run)/.
